packages/logs/README.md

```diff
@@ -1,6 +1,6 @@
 # Browser Log Collection
 
-Send logs to Datadog from web browsers or other Javascript clients with the browser logs SDK.
+Send logs to Datadog from web browsers with the browser logs SDK.
 
 With the browser logs SDK, you can send logs directly to Datadog from JS clients and leverage the following features:
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏it probably worth removing other `JS clients` mentions

Reply:
Good catch. I spotted only this other one